_quarto.yml

```
@@ -53,9 +64,10 @@ website:
         - about.qmd
         - privacy.qmd
 
-    - title: "Guides"
+    - title: "Docs"
       style: "docked"
       contents:
+        - docs-home.qmd
         - add-to-path.qmd
         - section: "User Interface"
```

```suggestion
        - section: "Getting Started"
          contents:
            - start.qmd
            - download.qmd
            - extensions.qmd
            - updating.qmd
            - add-to-path.qmd
        - section: "User Interface"
```

Let's use the current `start.qmd` as the landing page for "Guides" and combine the current content with some of what you think is most important from the new `docs-home.qmd`. We need to make sure all the files currently docked under "Start" get added to different sections that are navigable.